wikipron/scrape.py

```diff
@@ -24,15 +26,15 @@
 }
 
 
-def _skip_word(word: str, skip_spaces: bool) -> bool:
+def _skip_word(word: str, skip_spaces: bool, language: str) -> bool:
     # Skips reconstructions.
     if word.startswith("*"):
         return True
     # Skips multiword examples.
     if skip_spaces and (" " in word or "\u00A0" in word):
         return True
     # Skips examples containing a dash.
-    if "-" in word:
+    if "-" in word and language != "Min Nan":
```

Comment:
Could you leave a TODO here? We ought to add this to the config setup someday, methinks.